Only display 'pitch' link on campaign nodes.

angaither opened the pull request:
Remove title from pitch link.
Fixes #388

angaither committed:
Only display 'pitch' link on campaign nodes.
Remove title from pitch link.
Fixes #338

aaronschachter commented:
This works for changing the tab title, but for the visibility we'll need to revisit the access callback for node/*/pitch.  Ideally the check for node type should be done in a new function, like `dosomething_campaign_pitch_access`, so that anon or auth users can't hit the /pitch by just going to the URL.

angaither committed:
Updated pitch-page access.
- only allow admin/editors to access pitch page
- only return pitch page on campaign nodes

angaither committed:
Created a function in ds_user to determine staff member.

aaronschachter commented:
:+1: :+1: :+1:
